### PR TITLE
Import standalone application

### PR DESCRIPTION
**Pull request by roel-devries**
- Re-use WorkflowCommands from GOB-Workflow 
- Optionally add messagebroker logging handler
- Log default to stdout
- Optionally give destination path for content output

Also fixed:
- replace xlrd (not maintained) with openpyxl
- Use read() on get_object from objectstore
- add typing to logging.py

### CODE REVIEW

File: gobcore/logging/logger.py [hunk 1]
```diff
@@ -216,14 +221,17 @@ def _log(self, level, msg, kwargs=None):
         :param kwargs:
         :return: None
         """
-        logger = getattr(Logger._logger[self._name], level)
+        level_logger = getattr(self.log_instance, level)
         assert logger, f"Error: invalid logging level specified ({level})"
```
Comment (RoelWKramer) [inline gobcore/logging/logger.py:225]: assert level_logger here (if this didnt fail it is also not properly tested)

File: gobcore/utils.py [hunk 2]
```diff
@@ -130,5 +130,5 @@ def get_filename(name, offload_folder):
     dir = os.path.join(GOB_SHARED_DIR, offload_folder)
```
Comment (RoelWKramer) [inline gobcore/utils.py:130]: path = Path(GOB_SHARED_DIR, offload_folder)
path.mkdir(exist=True, parents=True)
return str(path)

Mix path and os.path.

Comment (RoelWKramer) [reply gobcore/utils.py:130]: Also if the function is called get_filename but it also creates a parent dir, maybe give it a better name, or split functionality?

Where this is used:
bla = self.get_filename()  # return type Path instead
bla.mkdir(exist_ok=True, parents=True)

File: tests/gobcore/workflow/test_start_commands.py [hunk 3]
```diff
@@ -191,3 +192,163 @@ def test_get_all(self, mock_load, mock_open, mock_start_command):
         start_commands.commands = MagicMock()
 
         self.assertEqual(start_commands.commands, start_commands.get_all())
+
+
+class Struct:
```
Comment (RoelWKramer) [inline tests/gobcore/workflow/test_start_commands.py:197]: Poor choice of type name? This is not a struct.
I think that argparse calls it a namespace.
https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.parse_args
https://docs.python.org/3.10/library/argparse.html#argparse.Namespace

**Review by RoelWKramer — COMMENTED** [review]
generally looks good, a few nitpicks

File: gobcore/logging/logger.py (hunk 1 above)
Comment (roel-devries) [reply gobcore/logging/logger.py]: fixed

**Review by RoelWKramer — DISMISSED** [review]

**Review by RoelWKramer — APPROVED** [review]

**Pull request merged**